File: crates/dashboard-model/src/victron_controller/dashboard/from_0_1_0_schedule_spec.rs


pub fn convert__schedule_spec__from__0_1_0(from: &crate::victron_controller::dashboard::v0_1_0::schedule_spec::ScheduleSpec) -> crate::victron_controller::dashboard::schedule_spec::ScheduleSpec {
    crate::victron_controller::dashboard::schedule_spec::ScheduleSpec {
        start_s: from.start_s.clone(),
        duration_s: from.duration_s.clone(),
        discharge: from.discharge.clone(),
        soc: from.soc.clone(),
        days: from.days.clone(),
    }
}

File: web/src/model/victron_controller/dashboard/v0_2_0/from_0_1_0_schedule-spec.ts
// @ts-nocheck
import {ScheduleSpec as v0_1_0_ScheduleSpec} from '../v0_1_0/ScheduleSpec'
import {ScheduleSpec as v0_2_0_ScheduleSpec} from './ScheduleSpec'

export function convert__schedule_spec__from__0_1_0(from: v0_1_0_ScheduleSpec): v0_2_0_ScheduleSpec {
    return new v0_2_0_ScheduleSpec (
        from.start_s,
        from.duration_s,
        from.discharge,
        from.soc,
        from.days
    )
}